source/guides/network/creating-a-private-test-net.rst

```diff
@@ -5,165 +5,232 @@
+
+This will create a folder called ``target`` by default (It can be changed with the ``-t`` option). Inside there's a folder called ``config`` containing, among other things, the generated complete configuration file (``target/config/preset.yml``) ready to be used to build the network.
 
-3. Display the content of the ``addresses.yaml`` file.
+********************
+Building the network
+********************
+
+This command prepares the necessary Docker files based on the provided configuration:
 
 .. code-block:: bash
 
-    cat addresses.yaml
+    symbol-bootstrap compose
+
+It only needs to be run once.
 
-4. Under the section ``nemesis_addresses``, you will find the key pairs which contain |privatenetworkcurrency|.
-Copy the private key of the first account.
+****************
+Running the node
+****************
 
-5. Type the command ``symbol-cli profile import`` using the key obtained in the previous step.
+Finally, execute this command to start the necessary Docker instances and boot your network:
 
 .. code-block:: bash
 
-    symbol-cli profile import
+    symbol-bootstrap run
+
+Stop the process by pressing ``Ctrl+C``.
 
-    Enter network type (MIJIN_TEST, MIJIN, MAIN_NET, TEST_NET): TEST_NET
-    Enter your private key: 123***456
-    Enter a Symbol Node URL. (Example: http://localhost:3000): http://localhost:3000
-    Insert profile name: base-profile
+.. note:: To run the docker containers in the background of your terminal, you can run the service in detached mode using the option ``--detach`` or ``-d``.
+
+**********************
+The all-in-one command
+**********************
 
-You should see the account credentials in your terminal.
+The above three commands (``config``, ``compose`` and ``run``) can be merged into one:
 
 .. code-block:: bash
 
-    Profile stored correctly
-    ┌─────────────┬──────────────────────────────────────────────────────────────────┐
-    │ Property    │ Value                                                            │
-    ├─────────────┼──────────────────────────────────────────────────────────────────┤
-    │ Address     │ SCVG35-ZSPMYP-L2POZQ-JGSVEG-RYOJ3V-BNIU3U-N2E6                   │
-    ├─────────────┼──────────────────────────────────────────────────────────────────┤
-    │ Public Key  │ 654...321                                                        │
-    ├─────────────┼──────────────────────────────────────────────────────────────────┤
-    │ Private Key │ 123...456                                                        │
-    └─────────────┴──────────────────────────────────────────────────────────────────┘
-
-As the name suggests, the **private key has to be kept secret at all times**.
-Anyone with access to the private key ultimately has control over the account.
-On the other hand, you can share securely the public and address of your account with other participants of the network to receive transactions from them.
+    symbol-bootstrap start -p <preset> -a <assembly>
```

Comment:
For this guide maybe focus the command to `symbol-bootstrap start -p bootstrap`?

Reply:
Yes. And the "running a private node" guide will focus on `symbol-bootstrap start -p testnet`.